components/image-builder-mk3/pkg/orchestrator/monitor.go

```diff
@@ -232,7 +232,7 @@ func extractBuildResponse(status *wsmanapi.WorkspaceStatus) *api.BuildResponse {
 		info = extractBuildStatus(status)
 		msg  = status.Message
 	)
-	if status.Phase == wsmanapi.WorkspacePhase_STOPPING {
+	if status.Phase == wsmanapi.WorkspacePhase_STOPPING || status.Phase == wsmanapi.WorkspacePhase_STOPPED {
```

Comment:
this also fixes image builds to report the right failure message. In mk2, image builds immediately transition to `Stopped`, skipping `Stopping`, as there's nothing to dispose